frontend/src/v5/ui/controls/assigneesSelect/assigneesSelect.styles.ts

```diff
@@ -58,3 +65,23 @@ export const AssigneesListContainer = styled.div`
 		}
 	}
 `;
+
+export const AddUserButton = styled(PopoverCircle).attrs({
+	size: 'small',
+})`
+	padding: 5px;
+	box-sizing: border-box;
+	color: ${({ theme }) => theme.palette.base.main};
+	&& {
+		border: 1px dashed ${({ theme }) => theme.palette.base.light};
+		outline: none;
+		margin: 0;
+		&::before {
+			background-color: transparent;
+		}
+	}
+`;
+
+export const Tooltip = styled(TooltipBase)`
+	margin: 0 0 0 13px;
```

Comment:
![image](https://github.com/3drepo/3drepo.io/assets/43547703/66d0937d-508a-4cb2-828b-0d236b2d3e99)
The component is now misplaced and you can click on the red cross and it will still open.
This is probably due to the fact that you would be clicking on the margin, which is part of the component, even though you are not hovering it.

You can fix that by replacing the style of the tooltip to 
```ts
margin: 0;
width: 24px; // so that the tooltip has the same width of the button
```